src/rngs/entropy.rs

```diff
@@ -30,9 +30,17 @@ use rngs::{OsRng, JitterRng};
 /// external entropy then primarily use the local PRNG ([`thread_rng`] is
 /// provided as a convenient, local, automatically-seeded CSPRNG).
 ///
+/// # Panics
+///
+/// In the extraordinary situation that both [`OsRng`] and [`JitterRng`] fail,
```

Comment:
I think for now we don't even define `EntropyRng` in no-std mode, though we do for WASM, which doesn't always have a working entropy source. I.e. 'extraordinary' may need some qualification.